mavis/python/mavis_tacplus_keycloak.py

```
@@ -0,0 +1,294 @@
+
+def decode_token_claims(token_data):
+	"""Decode access token JWT and return claims dict, or None on failure."""
+	if not isinstance(token_data, dict) or "access_token" not in token_data:
+		print(
+			"mavis_tacplus_keycloak: token response missing 'access_token' key",
+			file=sys.stderr,
+		)
+		return None
+	try:
+		return decode_jwt_payload(token_data["access_token"])
+	except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)